lldb/source/Utility/FileSpec.cpp

```diff
@@ -118,53 +118,55 @@ bool needsNormalization(const llvm::StringRef &path) {
     return true;
   for (auto i = path.find_first_of("\\/"); i != llvm::StringRef::npos;
        i = path.find_first_of("\\/", i + 1)) {
-    const auto next = safeCharAtIndex(path, i+1);
+    const auto next = safeCharAtIndex(path, i + 1);
```

Comment:
I think these are all unrelated changes? Please use `git clang-format` to only format the lines changed in the PR.